api/justfile

```
@@ -121,6 +121,11 @@ doc-test: wait-up
       --fail \
       'http://localhost:50280/v1/schema/' > openapi.json
 
+# Run the API schema tests using schemathesis
+test-schema: wait-up
+    just doc-test
```

`doc-test` can be changed to a pre-requisite recipe here because we depend on the output of that recipe `openapi.json` to run this step.
```suggestion
test-schema: wait-up doc-test
```